manticore/ethereum.py

```diff
@@ -173,6 +178,9 @@ def __build_source_map(self, bytecode, srcmap):
     def runtime_bytecode(self):
         # Removes metadata from the tail of bytecode
         end = None
+        if len(self._runtime_bytecode) == 0:
```

Comment:
maybe even .. if len(self._runtime_bytecode) < 44 ?
44 being the size of the metadata at the end of the bytecode

Reply:
great suggestion. I added this check to deal with empty `runtime_bytecode` generated for interface-like contracts. was unaware of the metadat length, will change it to be `< 44`